core/testcasecontroller/simulation/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=missing-module-docstring
+from .simulation import Simulation
```

Comment:
Is it better to use relative import?

Reply:
I'm sorry I don't quite understand how to fix this. Exactly, I don't know how to change to relative import.

In my code, the architecture of the `testcasecontroller/simulation` module is referenced to the architecture of the `testcasecontroller/algorithm` module.